ledger-api/grpc-definitions/com/daml/ledger/api/v1/active_contracts_service.proto

```diff
@@ -45,6 +45,10 @@ message GetActiveContractsRequest {
   // If not set the current ledger end offset will be used.
   // Optional
   string active_at_offset = 4;
+
+  // Must be a valid domain ID
```

Comment:
Do we want to document when it should/should not bet defined?

Reply:
Absolutely, I have a TODO item for that in the PR description :+1:

Reply:
closing comments related ACS/transfers as it is a WIP, need to be rereviewed